Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM registry.svc.ci.openshift.org/openshift/release:golang-1.15 AS builder
+FROM registry.svc.ci.openshift.org/openshift/release:golang-1.16 AS builder
```

Comment:
wrong commit; this should go in the "updating to use tag v0.21.0 for kubernetes repos like the rest of openshift" commit, not "Updating api to reduce the number of watch requests "

Reply:
done